### ISSUE TITLE
Call to a member function find() on null

### ISSUE THREAD

**Issue by ArrowBaZ** [getgrav/grav-plugin-form#96]
Hello, with following form code

```
form:
    name: Upload
    fields:
        custom_file:
          type: file
          label: A Label
          multiple: false
          destination: 'self@'
          random_name: false
          avoid_overwriting: false
          limit: 10
          filesize: 5
          accept:
            - image/*
```

each time i add image, i get error message on hover `Call to a member function find() on null`

![image](https://cloud.githubusercontent.com/assets/2211032/18700916/20d0bc2e-7fda-11e6-8ae4-9b567a0d89af.png)

**Comment by flaviocopes** [comment]
Can't replicate on my setup with the same form. 

You have the latest Grav and latest Admin and Form plugins?

**Comment by flaviocopes** [comment]
This is in Admin or in the frontend?

**Comment by ArrowBaZ** [comment]
This is on front page, yes fresh download.

**Comment by fredrikekelund** [comment]
I had the same issue, should be fixed by #105

**Comment by flaviocopes** [comment]
Thanks @fredrikekelund 👍
